### ISSUE TITLE
Another presolver crash with "Check failed: !DomainIsEmpty(ref)"

### ISSUE THREAD

**Issue by gregy4** [google/or-tools#1163]
I found same symptoms as described in #1136 issue but on build 7.0.6546 the problem is still present. My feasibility problem was without problem solved on previous stable version (6.10).

You can reproduce it with following code

```cpp
CpModel model = new CpModel();
InputStream inputStream =new FileInputStream("Model 1099.txt");
model.getBuilder().mergeFrom(inputStream);
CpSolver cpSolver = new CpSolver();
CpSolverStatus solverStatus = cpSolver.solve(model);
System.out.println(solverStatus);
```
[Model 1099.txt](https://github.com/google/or-tools/files/3021432/Model.1099.txt)

**Comment by lperron** [comment]
can you instead print the model in plain text ?

CpModel.proto().toString() ?
Laurent Perron | Operations Research | lperron@google.com | (33) 1 42 68 53
00

**Comment by lperron** [comment]
This is fixed on master.
It will be available in 7.1.

Thanks for the report.

Laurent Perron | Operations Research | lperron@google.com | (33) 1 42 68 53
00
